po/LINGUAS

```diff
@@ -1 +0,0 @@
-fr
```

Comment:
As already mentioned, please restore the LINGUAS files. It's fine to remove all current translations in there since they are useless anyway.

Reply:
Have added LINGUAS back.